semanticdb/integration/src/main/scala/example/ValPattern.scala

```diff
@@ -21,6 +21,25 @@ class ValPattern {
         rightVar
       )
     )
+    locally {
+      val (left, right) = (1, 2)
+      val Some(number1) =
+        Some(1)
+
+      var (leftVar, rightVar) = (1, 2)
+      var Some(number1Var) =
+        Some(1)
+      println(
+        (
+          number1,
+          left,
+          right,
+          number1Var,
+          leftVar,
+          rightVar
+        )
+      )
+    }
```

Comment:
The commit message has a typo in the desugaring listing. Should say `val b = x$1._2`.